pydantic_ai_slim/pydantic_ai/toolsets/combined.py

```diff
@@ -80,6 +80,7 @@ async def get_tools(self, ctx: RunContext[AgentDepsT]) -> dict[str, ToolsetTool[
                     args_validator=tool.args_validator,
                     source_toolset=toolset,
                     source_tool=tool,
+                    timeout=tool.timeout,
```

Comment:
I don't think we need it here as it's already on the `source_tool` that the `FunctionToolset` will see